api/v1alpha1/ioping_types.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2019 The xridge kubestone contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// IopingVolumeSpec contains the configuration for the volume that the ioping job
+// will use for benchmarking
+// TODO: Factor VolumeSpec into a common one
+type IopingVolumeSpec struct {
```

Comment:
This is exactly the same as FioVolumeSpec. Should be moved to k8s_types.

Reply:
👍